### ISSUE TITLE
LineLengthExtendedCheck: java.lang.NoSuchMethodError in Sonar Checkstyle plugin v2.3

### ISSUE THREAD

**Issue by rdiachenko** [sevntu-checkstyle/sevntu.checkstyle#366]
The following exception occurred when LineLengthExtendedCheck is turned on in Sonar:

```
[WARN] [04:51:27.061] /A.java: Got an exception - java.lang.NoSuchMethodError: com.puppycrawl.tools.checkstyle.Utils.lengthExpandedTabs(Ljava/lang/String;II)I
[ERROR] [04:51:27.062] Exception occurred during the analysis of file /A.java .
java.lang.NoSuchMethodError: com.puppycrawl.tools.checkstyle.Utils.lengthExpandedTabs(Ljava/lang/String;II)I
at com.github.sevntu.checkstyle.checks.sizes.LineLengthExtendedCheck.finishTree(LineLengthExtendedCheck.java:239)
at com.puppycrawl.tools.checkstyle.TreeWalker.notifyEnd(TreeWalker.java:430)
at com.puppycrawl.tools.checkstyle.TreeWalker.walk(TreeWalker.java:386)
at com.puppycrawl.tools.checkstyle.TreeWalker.processFiltered(TreeWalker.java:216)
at com.puppycrawl.tools.checkstyle.api.AbstractFileSetCheck.process(AbstractFileSetCheck.java:79)
at com.puppycrawl.tools.checkstyle.Checker.process(Checker.java:271)
at org.sonar.plugins.checkstyle.CheckstyleExecutor.execute(CheckstyleExecutor.java:85)
at org.sonar.plugins.checkstyle.CheckstyleSensor.analyse(CheckstyleSensor.java:57)
at org.sonar.batch.phases.SensorsExecutor.executeSensor(SensorsExecutor.java:79)
at org.sonar.batch.phases.SensorsExecutor.execute(SensorsExecutor.java:70)
at org.sonar.batch.phases.PhaseExecutor.execute(PhaseExecutor.java:119)
at org.sonar.batch.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:194)
at org.sonar.api.platform.ComponentContainer.startComponents(ComponentContainer.java:93)
at org.sonar.api.platform.ComponentContainer.execute(ComponentContainer.java:78)
at org.sonar.batch.scan.ProjectScanContainer.scan(ProjectScanContainer.java:233)
```

Current Checkstyle plugin for Sonar v2.3 uses Checkstyle 6.4.1 (http://docs.sonarqube.org/display/PLUG/Checkstyle+Plugin) but in Checkstyle 6.5 the backward compatibility was broken (http://checkstyle.sourceforge.net/releasenotes.html#Release_6.5). As a result the aforementioned exception occurred.

**Comment by rdiachenko** [comment]
The issue was raised in SonarQube's mailing list: https://groups.google.com/forum/#!topic/sonarqube/we1uaK570m8

**Comment by igieon** [comment]
This is probably solved with #379 .

**Comment by romani** [comment]
It could be resolved after Sonar team update their plugin. Can make +1 at mail list ? To remind them about this issue.

**Comment by romani** [comment]
will be resolved by  #401
